apps/meeting/src/views/DeviceSetup/index.tsx

```diff
@@ -6,15 +6,22 @@ import { Heading } from 'amazon-chime-sdk-component-library-react';
 import MeetingJoinDetails from '../../containers/MeetingJoinDetails';
 import { StyledLayout } from './Styled';
 import DeviceSelection from '../../components/DeviceSelection';
+import { useAppState } from '../../providers/AppStateProvider';
+import DefaultDevicesSelection from '../../containers/DefaultDevicesSelection';
 
-const DeviceSetup: React.FC = () => (
-  <StyledLayout>
+const DeviceSetup: React.FC = () => {
+  const { skipDeviceSelection } = useAppState();
+
+  return (
+    <StyledLayout>
     <Heading tag="h1" level={3} css="align-self: flex-start">
       Device settings
     </Heading>
+    {skipDeviceSelection && <DefaultDevicesSelection />}
```

Comment:
This flow does not make sense to me. Because when selecting "skip device selection" checkbox, users expect to handle the device selection manually. But this `DefaultDevicesSelection` select the first device for audio and video automatically. It makes this flow identical with not selecting "skip device selection".

Suggest removing this line as well as `DefaultDevicesSelection`. We can manually select devices 1. in Device Setup page via device drop down or 2. in Meeting page via Meeting control.